examples/units_and_coordinates/venus_transit.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+"""
+==============================================
+Overplotting the position of the Venus transit
+==============================================
+
+How to accurately plot the position of Venus as it transitted in front
+of the Sun as observed by AIA.
+"""
+import matplotlib.pyplot as plt
+
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+from astropy.coordinates import solar_system_ephemeris
+from astropy.utils.data import download_file
+
+import sunpy.map
+from sunpy.coordinates import frames, get_body_heliographic_stonyhurst
+
+###############################################################################
+# Let's download an image of the Venus transit.
+f = download_file('http://jsoc.stanford.edu/data/events/Venus_AIA24s_1600/Out/fits/20120606_040731_UTC.0041.fits')
+aiamap = sunpy.map.Map(f)
+
+###############################################################################
+# For this example, we require high precision ephemeris information. The built-in
+# ephemeris provided by astropy are not accurate enough.
+solar_system_ephemeris.set('jpl')
```

Comment:
* You should add a warning about the download that is triggered.
* You should probably instead specify `de432s` for a more palatable ~10 MB download rather than a >100 MB download.
* Note that this download will happen every time the gallery examples are tested in CI.

Reply:
We should both cache this on Circle, and not download 100Mb if we can avoid it.